data/mods/DinoMod/egg.json

```
@@ -0,0 +1,108 @@
+[
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_dino",
+    "name": "dinosaur egg",
```

```suggestion
    "name": { "str": "dinosaur egg" },
```
Please use this format throughout.